extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/jsonb/PanacheMongoJsonbContextResolver.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.mongodb.panache.jsonb;
+
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+import javax.json.bind.JsonbConfig;
+import javax.ws.rs.ext.ContextResolver;
+import javax.ws.rs.ext.Provider;
+
+/**
+ * This will provides serialization/deserialization of a MongoDB ObjectId as a String.
+ *
+ * Note: to avoid automatically installing it, it is removed from the index via an exclusion on the Jandex Maven plugin.
```

Comment:
Maybe explaining how it is registered in this comment would be useful?

Reply:
Done.
Be aware that it will be improved it's part of my todo list